The Add-AzureRmApplicationGatewayRedirectConfiguration was missing.

surajmb opened the pull request:
The Add-AzureRmApplicationGatewayRedirectConfiguration was missing in the redirect configuration creation in the path based redirect.

azurecla commented:
Hi __@ms-prkhad__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@ms-prkhad Thanks for the contribution! I've forwarded this to the author.

tamram commented:
That's correct, thanks. Ready to merge.

surajmb committed:
The Add-AzureRmApplicationGatewayRedirectConfiguration was missing.

The Add-AzureRmApplicationGatewayRedirectConfiguration was missing in the redirect configuration creation in the path based redirect.
